packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -5,12 +5,13 @@ import { FormattedMessage } from "react-intl";
 
 import { ClearInputAdornment } from "../common/ClearInputAdornment";
 import { AsyncOptionsProps } from "../hooks/useAsyncOptionsProps";
+import { messages } from "../messages";
 
 export interface FinalFormSelectProps<T> extends FieldRenderProps<T, HTMLInputElement | HTMLTextAreaElement> {
     getOptionLabel?: (option: T) => string;
     getOptionValue?: (option: T) => string;
     children?: React.ReactNode;
-    clearable?: boolean;
```

Comment:
This is a breaking change and requires a changeset

Reply:
Should there be two changesets? One for the removal of `clearable` and one for adding the placeholder item?

Reply:
You could add a second line to the existing changeset regarding the new behavior with the empty option.